libs/awscdk/src/app.ts

```diff
@@ -167,47 +129,6 @@ export class App extends core.App {
     return this.synthedOutput;
   }
 
-  protected typeForFqn(fqn: string): any {
```

Comment:
This is going to break users who leveraged this method to create a custom app

@hasanaburayyan ☝️

Reply:
Yea the other break would be in the `AppProps` interface where the factory is required.

Reply:
I've added notes about breaking changes to the PR description.